pom.xml

```diff
@@ -30,8 +30,15 @@
     <properties>
         <powermock.version>1.6.4</powermock.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <workflow.version>1.10</workflow.version>
+        <workflow-step-api.version>2.9</workflow-step-api.version>
+        <workflow-aggregator.version>2.5</workflow-aggregator.version>
+        <workflow-job.version>2.11</workflow-job.version>
         <jenkins.version>2.0</jenkins.version>
+        <workflow-api.version>2.11</workflow-api.version>
+        <workflow-cps.version>2.29</workflow-cps.version>
+        <workflow-durable-task-step.version>2.11</workflow-durable-task-step.version>
```

Comment:
They are - none of the core Pipeline plugins have dependencies on core >=2.0 until `workflow-job` 2.12.